pkg/web/static/js/transactionInfo.js

```diff
@@ -69,10 +94,35 @@ document.body.addEventListener('htmx:afterRequest', (e) => {
     transactionRejectForm.reset()
     setSuccessToast('Success! The secure envelope has been rejected.')
   }
+
+  if (e.detail.requestConfig.path === transactionSendEP && e.detail.requestConfig.verb === 'post' && e.detail.successful) {
+    const transactionCompleteForm = document.getElementById('transaction-complete-form');
+    const transactionCompleteModal = document.getElementById('transaction_complete_modal');
+    transactionCompleteModal.close();
+    transactionCompleteForm.reset();
+    enableSubmitBtn();
+    setSuccessToast('Success! The on-chain transaction has been sent to the counterparty.');
+  };
 });
 
 // Display success toast message if user is redirected to info page after accepting a transaction.
 const transactionSend = Cookies.get('transaction_send_success')
 if (transactionSend === 'true') {
   setSuccessToast('Success! The secure envelope has been accepted.')
-}
\ No newline at end of file
+}
+
+function disableSubmitBtn() {
```

Comment:
The disable and enable functions in the `main.js` file didn't work with the `Complete` button, so they've been added separately here. I'll have to investigate another time why the functions in main don't always execute as intended.